extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+.i-amp-story-quiz-container {
+    align-self: start;
+    --correct-color: #5BBA74 !important;
+    --correct-color-shaded: #2D8E42 !important;
+    --incorrect-color: #F34E4E !important;
+    --incorrect-color-shaded: #cc4141 !important;
+    --accent-color: #005AF0;
+    font-family: 'Poppins-bold', sans-serif !important;
+    background: var(--accent-color) !important;
+    border-radius: 32px !important;
+}
+
+.i-amp-story-quiz-prompt-container {
+    display: flex !important;
+    justify-items: center !important;
+    padding: 20px !important;
+    color: white !important;
+}
+
+.i-amp-story-quiz-prompt {
+    margin: 0px !important;
+}
+
+h1.i-amp-story-quiz-prompt {
+    font-size: 28px !important;
+    line-height: 36px !important;
+}
+
+h2.i-amp-story-quiz-prompt {
+    font-size: 22px !important;
+    line-height: 28px !important;
+}
+
+h3.i-amp-story-quiz-prompt {
+    font-size: 18px !important;
+    line-height: 24px !important;
+}
+
+.i-amp-story-quiz-option-container {
+    display: flex !important;
+    flex-direction: column;
+    background-color: white !important;
+    border-radius: 32px !important;
+    padding: 8px 8px 0px 8px !important;
+}
+
+.i-amp-story-quiz-option {
+    display: flex !important;
+    justify-items: start !important;
+    align-items: center !important;
+    border-radius: 32px !important;
+    padding: 8px 16px 8px 8px !important;
+    background-color: inherit !important;
+    margin-bottom: 8px !important;
+    color: #5F6368 !important;
+    border: solid 1px #DADCE0 !important;
+    /* TODO: Why won't this notation work? */
+    /* font: 16px/20px !important; */
```

Comment:
Tbh I didn't even know about this shorthand. If it doesn't work, can you remove the comment?